plutus-core/plutus-ir/src/PlutusIR/Purity.hs

```diff
@@ -9,28 +9,30 @@
 {-# LANGUAGE ViewPatterns          #-}
 
 module PlutusIR.Purity
-    ( isPure
-    , isSaturated
-    , isWorkFree
-    , EvalOrder
-    , unEvalOrder
-    , EvalTerm (..)
-    , Purity (..)
-    , termEvaluationOrder
-    ) where
+  ( isPure
+  , isSaturated
+  , isWorkFree
+  , EvalOrder
+  , unEvalOrder
+  , EvalTerm (..)
+  , Purity (..)
+  , termEvaluationOrder
+  ) where
 
-import PlutusCore.Builtin
-import PlutusCore.Pretty
-import PlutusIR
-import PlutusIR.Contexts
-
-import Control.Lens hiding (Strict)
+import Control.Lens ((^.))
 import Data.DList qualified as DList
 import Data.List.NonEmpty qualified as NE
+import PlutusCore.Builtin (BuiltinMeaning (..), ToBuiltinMeaning (..), TypeScheme (..))
```

Comment:
I don't think we reached any agreement on whether this is desirable or not. For me, this makes things worse, since these are project modules that I would expect people to be familiar with.

Reply:
> these are project modules that I would expect people to be familiar with.

Actually, in my case it is other way around: I might be acquainted with the `Prettyprinter` symbols from using the library in other projects, but I don't yet know all the symbols exported by our `PlutusTx` modules. 🤷🏼‍♂️ 

Anyway, I am not particularly hang up on explicit imports, I can undo those ones you prefer to leave wildcard.

Reply:
I agree that we should have a policy. I personally prefer explicit export lists, and I _think_ this can be enforced somewhere either in the formatter (once we decide what we want) or in the commit pipeline. Maybe we should talk about this during the technical discussion?